### PR TITLE
Preparing release v1.38.0

### PR DESCRIPTION
**Pull request by AlexAPB**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.38.0] - 2019-05-20
+### Changed
+- The Thrift encoding attempts to close request buffers immediately after
```
Comment (prashantv) [inline CHANGELOG.md]: s/buffers/body

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
...
+  reading the request bytes. This significantly reduced TChannel/Thrift memory
+  usage in some tests.
```
Comment (prashantv) [inline CHANGELOG.md]: maybe s/tests/scenarios

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.38.0] - 2019-05-20
+### Changed
+- The Thrift encoding attempts to close request buffers immediately after
+  reading the request bytes. This significantly reduced TChannel/Thrift memory
```
Comment (abhinav) [inline CHANGELOG.md]:
```suggestion
  reading the request bytes. This significantly reduces TChannel/Thrift memory
```

File: CHANGELOG.md (hunk 2 above)
Comment (abhinav) [inline CHANGELOG.md]:
```suggestion
  usage in some scenarios.
```

File: CHANGELOG.md (hunk 1 above)
Comment (abhinav) [inline CHANGELOG.md]:
```suggestion
- The Thrift encoding attempts to close request body immediately after
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.38.0] - 2019-05-20
+### Changed
+- The Thrift encoding attempts to close request body immediately after
```
Comment (abhinav) [inline CHANGELOG.md]:
```suggestion
- The Thrift encoding attempts to close the request body immediately after
```

**Comment by AlexAPB** [comment]
Thank you grammar friends 💃 🕺. I'll wait for a green build, before finishing the release.

**Pull request merged**